src/main/java/com/g4s8/ghman/web/TkApp.java

```diff
@@ -56,7 +57,7 @@
                                 "code", ".+",
                                 new TkGitHubAuthorization(data)
                             ),
-                            new FkGitHubAuthRedirection()
+                            new FkGitHubAuthRedirection(new EnvironmentVariables())
```

Comment:
@olenagerasimova `TkApp` should be the one taking the `EnvironmentVariables` object and passing it to `FkGitHubAuthRedirection` (and other objects if needed)